docs/run_test/platform.md

```diff
@@ -0,0 +1,163 @@
+# Run LISA
+
+- [Quick start](#quick-start)
+  - [Run in Azure](#run-in-azure)
+  - [Run in Ready computers](#run-in-ready-computers)
+- [Run Microsoft tests](#run-microsoft-tests)
+- [Run legacy LISAv2 test cases](#run-legacy-lisav2-test-cases)
+
+## Quick start
+
+`lisa.sh` (for Linux) and `lisa.cmd` (for Windows) are provided to wrap `Poetry`
+for you to run LISA test.
+
+In Linux, you could create an alias for this simple script. For example, add
+below line to add to `.bashrc`:
+
+```bash
+alias lisa="./lisa.sh"
+```
+
+If no argument specified, LISA will run some sample test cases with the default
+runbook (`examples/runbook/hello_world.yml`) on your local computer. You can use
+this way to verify your local LISA environment setup. This test will not modify
+your computer.
+
+```bash
+lisa
+```
+
+If you see any error from this run, please check [FAQ and
+troubleshooting](troubleshooting.md).
+
+### Run in Azure
+
+Please follow the steps below to configure your local computer to run LISA test
+against Linux VM on Azure.
+
+1. Sign in to Azure
+
+    Make sure [Azure
+    CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) or [Azure
+    PowerShell](https://docs.microsoft.com/en-us/powershell/azure/install-az-ps)
+    has been installed on your local computer. Then, please login to your Azure
+    subscription to authenticate your current session. LISA also supports other
+    Azure authentications, refer to [runbook reference](runbook.md).
+
+    Here, let's choose `Azure CLI` for the setup. You should see a page pop up
+    and all your Azure subscriptions shown in console.
+
+    ```bash
+    az login
+    ```
+
+2. Get the subscription id
+
+    LISA needs to know the Azure subscription ID for your testing. Run below
+    command to retrieve subscription information.
+
+    ```bash
+    az account show --subscription "<your subscription Name>"
+    ```
+
+    For now you only need the `<subscription id>` for future use.
+
+    ```json
+    {
+        "environmentName": "AzureCloud",
+        "homeTenantId": "<tenant id>",
+        "id": "<subscription id>",
+        "isDefault": true,
+        "managedByTenants": [],
+        "name": "<subscription name>",
+        "state": "Enabled",
+        "tenantId": "<tenant id>",
+        "user": {
+            "name": "<user account>",
+            "type": "user"
+        }
+    }
+    ```
+
+    AzureCloud in this example is set to default. You do not need to do that as
```

Comment:
This sentence is confusing. It needs to explain, the subscription, which is taken, will be used to run tests. So user needs to make sure it's a subscription for testing, not use subscription for production or other purposes.

"AzureCloud in this example is set to default."